src/nsidc/metgen/metgen.py

```diff
@@ -289,10 +294,19 @@ def checksum(file):
     return sha256.hexdigest()
 
 def s3_url(mapping, name):
-    mapping['s3_name'] = name
-    template = Template('s3://nsidc-cumulus-${environment}-ingest-staging/external/${auth_id}/${version}/${uuid}/${s3_name}')
+    """
```

Comment:
See comment above. We don't want to make Ops users type in the default value when creating an ini file.

Reply:
I've reworked the default values, added tests, and set the default bucket & kinesis stream names to the cumulus patterns.